code/_onclick/hud/parallax.dm

```diff
@@ -45,12 +45,14 @@ var/list/parallax_icon[(GRID_WIDTH**2)*3]
 	0,0,0,0,
 	0,0,0,0,
 	0,0,0,0,
-	1,1,1,1,
-	0,0,0,0)
+	0,0,0,0,
```

Comment:
Because it uses constant values now instead of relying the fact that the space sprite has max alpha.